ui/stories/bubble.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
```

Comment:
`import { storiesOf } from '../.storybook/facade'` to automatically get a basic verification test.

Reply:
Fixed